anta/tests/configuration.py

```diff
@@ -75,3 +76,55 @@ def test(self) -> None:
             self.result.is_success()
         else:
             self.result.is_failure(command_output)
+
+
+class VerifyRunningConfigLines(AntaTest):
+    """Verifies the given regular expression patterns are present in the running-config.
+
+    **Since this uses regular expression searches on the whole running-config, it can
+    drastically impact performance and should only be used if no other tests is available for.**
+    If possible, try using another Anta test that is more specific.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if all the patterns are found in the running-config.
+    * Failure: The test will fail if any of the patterns are NOT found in the running-config.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.configuration:
+      - VerifyRunningConfigLines:
+            regex_patterns:
+                - "^enable password.*$"
+                - "bla bla"
+    ```
+    """
+
+    name = "VerifyRunningConfigLines"
+    description = "Search the Running-Config for the given RegEx patterns."
+    categories: ClassVar[list[str]] = ["configuration"]
+    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaCommand(command="show running-config", ofmt="text")]
+
+    class Input(AntaTest.Input):
+        """Input model for the VerifyRunningConfigLines test."""
+
+        regex_patterns: list[str]
```

Comment:
```suggestion
        regex_patterns: list[RegexString]
```
See my comment below